src/fsharp/FSharp.Build/FSharp.Build.fsproj

```diff
@@ -2,10 +2,18 @@
 <!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Library</OutputType>
+  <PropertyGroup Condition="'$Configuration'=='Proto'">
```

Comment:
We shouldn't have to do this since `proto.proj` passes through a value.  Also, I don't like it.

Reply:
@brettfo yes indeed we do need to do it.  Here is build.proj from master branch.

https://github.com/dotnet/fsharp/blob/master/src/fsharp/FSharp.Build/FSharp.Build.fsproj#L7

In proto.proj we pass in TargetFramework, but there appear to be places that rely on TargetFramework and TargetFrameworks being consistent.  Regardless, without this filtering I couldn't figure out how to make proto Linux builds and desktop full product builds work successfully.  Perhaps when I get back from vacay we can sit down and look at the issues, and determine the right approach.  TBH, I had at least one version of this where the additional properties were not set in the proto, and I actually think that is probably the correct way to go, or maybe, having it set TargetFrameworks instead of TargetFramework.  However, there is not really a way of doing that for VisualFSharp.sln builds.

Kevin